hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/test/GenericTestUtils.java

```diff
@@ -245,6 +270,18 @@ public static File getTestDir() {
     return dir;
   }
 
+  /**
+   * Cleans-up the root directory from the property
+   * {@link #SYSPROP_TEST_DATA_DIR}.
+   *
+   * @return the absolute file of the test root directory.
+   */
+  public static File clearTestRootDir() {
```

Comment:
This is only called from setupTestRootDir(), so I don't think it needs to be a separate public function.  I would just do the fullyDelete() call in-line in setup TestRootDir.